tests/kernel/context/testcase.yaml

```diff
@@ -4,4 +4,4 @@ tests:
     min_ram: 20
     # FIXME: Remove `qemu_cortex_r5` exclusion once Zephyr SDK 0.11.3 is
     #        mainlined (see #22904).
-    platform_exclude: qemu_cortex_r5
+    platform_exclude: qemu_cortex_r5 nsim_hs_smp
```

Comment:
This is a big test to exclude, tests/kernel/context really needs to be passing.  If you absolutely need a workaround, can't you do a more targetted whitelisting, maybe with a ztest_test_skip() at the specific test case?

Reply:
make senses, i will change